pages/api/raw.ts

```diff
@@ -28,7 +28,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     return
   }
 
-  const { path = '/', odpt = '' } = req.query
+  const { path = '/', odpt = '', proxy = false, inline = false } = req.query
```

Comment:
remove the `inline=false` param, we won't need to modify PDF params here, it is not used anymore